### PR TITLE
Update rippled ubuntu tutorial to use cmake, not scons

### PR DESCRIPTION
**Pull request by jhaaaa**
Tested the updated steps using my existing ubuntu 16.04 environment (which I've used to install 0.90.0), but hoped to test on a clean environment. 

@mDuo13, would you mind testing out the updated instructions, please? And, thank you again for such a thorough and helpful JIRA ticket for this update!

### CODE REVIEW

**Review by mDuo13 — APPROVED** [review]
Fixed the problems I caught, should be good now

File: content/tutorials/manage-the-rippled-server/build-run-rippled-ubuntu.md [hunk 1]
```diff
@@ -59,51 +59,60 @@ These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the sof
...
+
+12. Use CMake to build a `rippled` binary executable from source code. The result will be a `rippled` binary executable in the `my_build` directory.
 
-    If this is your first time building `rippled,` you won't have a `build/` directory and can move on to the next step.
+      1. Generate the build system. Builds should be performed in a directory that is separate from the source tree root. In this example, we'll use a `my_build` directory that is a subdirectory of `rippled`.
 
-12. Build a `rippled` binary executable from source code. This may take about 20 minutes, depending on your hardware specs.
+              mkdir my_build
+              cd my_build
+              cmake -Dtarget=gcc.debug.unity ..
```
Comment (ximinez) [inline content/tutorials/manage-the-rippled-server/build-run-rippled-ubuntu.md]: 1. Would it be better to recommend building a release version?
2. PR https://github.com/ripple/rippled/pull/2579 is going to change the way these parameters work. I don't know the details, but this is an early warning.

**Review by mellery451 — APPROVED** [review]
seems fine - I echo what @ximinez said about 2579: it will impact these steps again in a few small ways.

File: content/tutorials/manage-the-rippled-server/build-run-rippled-ubuntu.md (hunk 1 above)
Comment (mDuo13) [reply content/tutorials/manage-the-rippled-server/build-run-rippled-ubuntu.md]: 1. Yeah, will add a step about checking out a release tag or something.
2. Noted.

Comment (mDuo13) [reply content/tutorials/manage-the-rippled-server/build-run-rippled-ubuntu.md]: After digging deeper, I noticed the instructions already include the step of checking out the master branch, but I added a step to confirm the commit ID. (See 7df5717 for that.)

**Comment by mellery451** [comment]
still 👍

**Pull request merged**